src/components/icons/Icon.vue

```diff
@@ -25,16 +30,48 @@ const path: ComputedRef<string | undefined> = computed(() => {
   }
   return found;
 });
+
+const iconColor: ComputedRef<string | undefined> = computed(
+  () => props.color && css[props.color]
+);
 </script>
 
 <template>
   <svg
-    class="remixicon"
+    :class="[css.remixicon, iconColor]"
```

Comment:
I think we don't need to create a new computed for `iconColor` (?). I think `css[color]` work just fine?

Reply:
it didn't because css is a Record, that requires a string key, but color can be `undefined`